### ISSUE TITLE
error message "MPU6050 connection failed"

### ISSUE THREAD

**Issue by borntoleave** [PetoiCamp/OpenCat-Old#10]
Hi, 
I have noticed some strange error message from the MPU6050 codes borrowed from the popular [i2cdevlib](https://github.com/jrowberg/i2cdevlib/tree/master/Arduino/MPU6050):
The [MPU6050 test code](https://github.com/PetoiCamp/OpenCat/tree/master/ModuleTests/testMPU) will occasionally show error message "MPU6050 connection failed", but the following codes can read all the sensor data just fine. And it seems to be a rare but [reproducible](https://community.cypress.com/thread/33264?start=0&tstart=0) error across several forums. 

Basically it's the function **mpu.testConnection()** always returns false. My current solution is just ignoring the error message since it's not affecting any later use on the sensor data. But does anyone have better insight why it is happening? Thanks!

**Comment by RongzhongLi** [comment]
Seems to be the problem of the gy-521.
